### PR TITLE
BaSP-MR-61: List-and-delete-added

### PR DESCRIPTION
**Pull request by Gonzarey96**
A list that allows deleting and obtaining detailed information about the table members.

### CODE REVIEW

**Comment by Federico-Peralta** [comment]
You have the name of the PR wrong

**Review by lorenzomauro59 — APPROVED** [review]
For me It's approved!

**Comment by fernandofernandezuy** [comment]
Good work Gonza!, It's a good practice to provide descriptions in our PRs

File: src/Components/Members/api.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    return data.data;
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+}
+
+export async function deleteMember(id) {
+  try {
+    const response = await fetch(`http://localhost:4000/api/members/${id}`, {
```
Comment (EmaBaldor) [inline src/Components/Members/api.js]: Environment variables should be used here.
process.env.REACT_APP_API_URL + `/members/${id}`

**Comment by Gonzarey96** [comment]
[[Members] List/Delete table](https://trello.com/c/FyRrK72k/61-members-list-delete-table)

**Comment by AleMac17** [comment]
[[Admin] List/Delete table](https://trello.com/c/EZavij9r/53-admin-list-delete-table)

File: src/Components/Members/api.js (hunk 1 above)
Comment (Gonzarey96) [reply src/Components/Members/api.js]: Changes made!

File: src/Components/Members/api.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      method: 'DELETE'
+    });
+    if (!response.ok) {
+      throw new Error('An error occurred while trying to delete the member');
+    }
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+}
```
Comment (LucianoAlarcon) [inline src/Components/Members/api.jsx]: Please change all to ES6

File: src/Components/Members/index.js [hunk 3]
```diff
@@ -1,25 +1,98 @@
...
-      });
+    const getMembers = async () => {
+      try {
+        const members = await fetchMembers();
+        setMembers(members);
+      } catch (error) {
+        console.error(error);
+        setError(error.message);
+      }
+    };
```
Comment (LucianoAlarcon) [inline src/Components/Members/index.js]: Move this function outside the useEffect

File: src/Components/Members/index.js [hunk 4]
```diff
@@ -1,25 +1,98 @@
...
+
+    getMembers();
   }, []);
 
+  const handleDelete = async (id) => {
+    setIsConfirmOpen(true);
+    setMemberToDelete(id);
+  };
```
Comment (LucianoAlarcon) [inline src/Components/Members/index.js]: Delete this async

**Review by LucianoAlarcon — APPROVED** [review]

**Review by LucianoAlarcon — APPROVED** [review]

**Pull request merged**